ui/webui/test/helpers/wizard.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+
+class Wizard():
```

Comment:
I wonder if we shouldn't try to be more abstract. Is it important for the test that this is a wizard with steps? Don't we just want to control the installation?

Here is an example of an anabot test. The example is extreme, but we could get closer to this level of abstraction:
https://github.com/rhinstaller/kickstart-tests/blob/master/anabot-1.xml

Reply:
This is hard to compare, because previously we were entering and leaving spokes as defined in the xml. That should be mostly also possible in the wizard, but we still want to have the option to go through it linearly, then go back etc. (and I guess that will be the default for most e2e tests).

Reply:
I just wanted to rename the classes to something more abstract (like `Installation` or `Installer` instead of `Wizard`). That's all.

Reply:
nothing against that, updating.